File: RoRClient/Models/Game/Coal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RoRClient.Models.Game
{
    /// <summary>
    /// Klasse für Ressource Kohle
    /// </summary>
    public class Coal : Resource, IPlaceableOnSquare
    {
        public Coal(Guid id, Square square) : base(id, square)
        {
            this.id = id;
        }
    }
}

File: RoRServer/src/main/java/models/game/Coal.java
package models.game;

/**
 * @author Andreas Pöhler, Isabell Rott, Juliane Lies
 * Klasse für Ressource Kohle
 */
public class Coal extends Resource {

	public Coal(String sessionName, Square square, int quantity) {
		super(sessionName, square, "Coal", quantity);
	}
	
	public Coal(String sessionName, int quantity) {
		super(sessionName, "Coal", quantity);
	}
}
